app/core/db.php

```diff
@@ -0,0 +1,23 @@
+// app/core/db.php
+<?php
+
+
+$host = 'mysql-blog'; // This should match the service name in your docker-compose.yml
+$db   = 'mydatabase'; // Database name
+$user = 'user'; // MySQL username
+$pass = 'userpassword'; // MySQL password
```

Comment:
Generally speaking we should avoid committing files that contain credentials.
I'd also suggest these could be kept in a `.env` file or the like.